src/components/svgs/stations/suzhourt-int.tsx

```diff
@@ -94,6 +94,7 @@ const SuzhouRTIntStation = (props: StationComponentProps) => {
                     style={{ cursor: 'move' }}
                 >
                     <rect
+                        id={`stn_core_${id}`}
```

Comment:
This won't work if users loosen the mouse on the below circle. A mask element is needed :)

Reply:
Oh, that's right.